### DISCUSSION TITLE
Who composed the 1812 Overture?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#849] [Galaxy Trivia] [answered]
**Bonus:** What unusual "instrument" was used in it?

**Comment by peterbaccaro** [comment, marked as answer]
The 1812 Overture was composed by **Pyotr Ilyich Tchaikovsky**, a renowned Russian composer. He composed the piece in 1880 to commemorate the Russian victory over Napoleon's forces in 1812 during the French invasion of Russia.
The 1812 Overture is known for its grand orchestration, including **cannons**, bells, and a prominent part for the Russian national anthem. It has become one of Tchaikovsky's most famous and frequently performed works.
Therefore the unusual "instrument" used in it is **cannons**.
